packages/react-ui-core/src/Icon/Icon.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react'
+import RPT from 'prop-types'
+
+export default class Icon extends Component {
```

Comment:
If this is only supporting SVG file, I really don't think it should be called `Icon` as the component name. @lfender6445 thoughts?